net-misc/econnman/econnman-1.1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{4..7} )
+
+inherit python-any-r1
+
+DESCRIPTION="ConnMan User Interface written with EFL & python"
+HOMEPAGE="https://phab.enlightenment.org/w/projects/econnman/ https://www.enlightenment.org/"
+SRC_URI="https://download.enlightenment.org/rel/apps/econnman/econnman-1.1.tar.xz"
```

Comment:
Why are you hardcoding the version? :-(

Reply:
Sigh, these fridays man :)

Started a new blank ebuild with Python Magic Tricks on my mind, forgot to change that. Will fix now.